components/Navbar/navbar.css

```diff
@@ -0,0 +1,78 @@
+@import "/default.css";
+
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+
+.input-for-nav {
+    border-radius: 4px;
+    border-style: none;
+    padding: .3rem;
+    outline: none;
+}
+
+.container {
+    width: 50rem;
+}
+
+/* **************************** standard **************************** */
+
+.header {
+    display: flex;
+    align-items: center;
+    background-color: var(--primary-color);
+    height: 2.8rem;
+    margin: 2rem;
+    color: #eee;
+    padding: 1rem;
+}
+
+.text {
+    margin-left: 2rem;
+    font-size: 1.2rem;
+    font-weight: 600;
+}
+
+.menu-container {
+    display: flex;
+    align-items: center;
+}
+
+.items {
+    margin-left: auto;
+}
+
+.items i{
```

Comment:
Instead of directly applying style to `i`  element you can give some class to it and then apply style on it.